docs/source/technical-overview.md

```diff
@@ -1,104 +1,134 @@
-## Technical Overview
+# Technical Overview
+
+The **Technical Overview** section gives you a high-level view of:
+
+- JupyterHub's Subsystems: Hub, Proxy, Single-User Notebook Server
+- how the subsystems interact
+- the process from JupyterHub access to user login
+- JupyterHub's default behavior
+- customizing JupyterHub
+
+The goal of this section is to share a deeper technical understanding of
+JupyterHub and how it works.
+
+## The Subsystems: Hub, Proxy, Single-User Notebook Server
 
 JupyterHub is a set of processes that together provide a single user Jupyter
-Notebook server for each person in a group.
+Notebook server for each person in a group. Three major subsystems are started
+by the `jupyterhub` command line program:
 
-### Three subsystems
-Three major subsystems run by the `jupyterhub` command line program:
+- **Hub** (Python/Tornado): manages user accounts, authentication, and
+  coordinates Single User Notebook Servers using a Spawner.
 
-- **Single-User Notebook Server**: a dedicated, single-user, Jupyter Notebook server is
-  started for each user on the system when the user logs in. The object that
-  starts these servers is called a **Spawner**.
 - **Proxy**: the public facing part of JupyterHub that uses a dynamic proxy
   to route HTTP requests to the Hub and Single User Notebook Servers.
-- **Hub**: manages user accounts, authentication, and coordinates Single User
-  Notebook Servers using a Spawner.
+  [configurable http proxy](https://github.com/jupyterhub/configurable-http-proxy)
+  (node-http-proxy) is the default proxy.
 
-![JupyterHub subsystems](images/jhub-parts.png)
+- **Single-User Notebook Server** (Python/IPython/Tornado): a dedicated,
```

Comment:
IPython is not involved here. Probably just "Python/Tornado" or "Python/Jupyter Notebook/Tornado".